### PR TITLE
Add sclite mode

### PR DESCRIPTION
**Pull request by desh2608**
SCLITE uses different costs for ins/del/sub = 3/3/4. This PR adds support for sclite mode.

### CODE REVIEW

**Comment by desh2608** [comment]
@pzelasko could you take a look when you have some time?

File: tests/test_align.py [hunk 1]
```diff
@@ -45,6 +45,17 @@ def test_edit_distance():
         'total': 2
     }
 
+def test_edit_distance_sclite():
```
Comment (pzelasko) [inline tests/test_align.py:48]: Can you adjust the test so that it clearly shows there’s a different result depending on the flag setting?

**Review by pzelasko — COMMENTED** [review]
Cool! I have one request otherwise LGTM. I am traveling right now and might merge and release on pypi after Thursday.

File: tests/test_align.py (hunk 1 above)
Comment (desh2608) [reply tests/test_align.py:48]: I am trying to come up with a simple example but can't think of anything. In this example itself, using uniform costs could give you `[(a,b),(b,c)]` or `[(a,*),(b,b),(*,c)]` both having cost=2, whereas sclite would always return the second alignment.

**Comment by pzelasko** [comment]
@csukuangfj since you contributed the adoption of CMake, do you have any idea why some MacOS builds did not work in this PR, and how to debug it?

**Comment by csukuangfj** [comment]
> @csukuangfj since you contributed the adoption of CMake, do you have any idea why some MacOS builds did not work in this PR, and how to debug it?

@pzelasko 

Please see #13

**Pull request merged**